jest.config.js

```diff
@@ -13,10 +13,15 @@ module.exports = {
     // Protons use a slightly different pattern because it's the only source pattern directory that
     // has a base-level index.js (this is how all components ensure access to base-level styles & variables).
     '^protons$': '<rootDir>/source/_patterns/00-protons/',
```

Comment:
Can we just remove this line?

Reply:
I think so, but I wasn't sure. Probably fine if you think we can

Reply:
Okay, no, we can't just remove it. 
`Cannot find module 'protons' from 'index.js'`
maybe a different regex pattern?

Reply:
Where does this error happen? I want a single "protons" in this list, not 2.

Reply:
removing that line makes 18 tests fail, so it's in a lot of places

Reply:
I'm experimenting with regex patterns to match both cases. If I'm right, we just need a regex pattern that matches these, right?:

* protons
* protons/
* protons/<thing>

Reply:
i.e. can we remove line 16, not line 15. the comment in line 13/14 explains why line 15 is needed

Reply:
This line should do it, I think? Double check @dslemay , @illepic :
`'^protons[/]?(.*)': '<rootDir>/source/_patterns/00-protons/$1',`

Reply:
Line 16 was needed to match protons/thing => protons/thing, specifically for the utilities.js file that is there. I think this regex pattern should match all our needs